README.md

```diff
@@ -47,47 +47,82 @@
 * [yup](https://github.com/jquense/yup)
 * [zod](https://github.com/vriad/zod)
 
-## Criteria
```

Comment:
Why did you get rid of the "Criteria" section?

Reply:
Maybe we can leave ### Validation in, but ### Interface felt like it had nothing to do with adding cases for packages per se.

Reply:
IMHO criteria + validity sections are important because only packages that match this interface are accepted for this benchmark.
The most important criteria seems to me "is able to infer types from the schema" part, maybe that should be added to the criteria section too.